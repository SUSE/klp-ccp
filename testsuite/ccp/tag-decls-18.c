enum B
{
	B_b,
};

void pu_f(enum B *)
{}

enum B;
