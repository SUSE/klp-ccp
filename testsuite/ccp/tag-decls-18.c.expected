/* klp-ccp: from testsuite/ccp/tag-decls-18.c */
enum B;

void klpp_pu_f(enum B *)
{}
